controllers/argocdexport/job.go

```diff
@@ -29,6 +29,7 @@ import (
 	argoprojv1alpha1 "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
 	argoproj "github.com/argoproj-labs/argocd-operator/api/v1beta1"
 	"github.com/argoproj-labs/argocd-operator/common"
+	"github.com/argoproj-labs/argocd-operator/pkg/argoutil"
 	util "github.com/argoproj-labs/argocd-operator/pkg/util"
```

Comment:
Better to make this a standard that to avoid renaming when not necessary.
```suggestion
	"github.com/argoproj-labs/argocd-operator/pkg/util"
```